### DISCUSSION TITLE
Workshop with 7 raw materials

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#514] [Q&A]
Is there any limitation, or any way to limit, the maximum number of raw materials that a workshop is storing?
I am building a Grand Temple which requires clay, at the same time the pottery industry needs some of the clay, so I would rather not stockpile.
Unfortunately, the cartmen are very adamant to only deliver clay to pottery workshops, and not at all fairly.
Some of my workshops have 1-2 stored clay, while I have workshops with 6 or even 7 clay stored.

Update: with stockpiling turned on, the warehouses closer to the GT and work camps are no longer Getting the clay.

And in the mean time, a Pottery Workshop across the street from 3 Clay Pits has 0 units stored, while another which is not that close to pits has 9 units stored.
Other workshops (Weapons, Oil) won't take more than 3 units each before the raw materials end up in warehouses.

It's a race against time for the Foreman to get to the clay faster than the warehouse can dispatch to the workshops.

**Comment by crudelios** [comment]
What version of augustus are you using? The latest unstable has some improvements to resource distribution.

**Reply by catalin-duta** [reply]
I'm on 3.0.1.282.

**Reply by crudelios** [reply]
This should have been fixed lately. I'm going to move this discussion to an issue. Can you post your savefile, please?

**Reply by catalin-duta** [reply]
I added two savefiles and some specific info.
I did already check the commit history, the more recent builds (after the android settings fix) don't seem to deal with resource distribution.

**Reply by crudelios** [reply]
It's this one: https://github.com/Keriew/augustus/commit/c1525793e8089787e0798c1cf574dacd009a779f

**Reply by catalin-duta** [reply]
I don't know the exact commit ID for 3.0.1.282 but I'm pretty sure it was after that one. I only used the stable build very briefly, because I was missing the pause button, which seems to have been merged from Julius to Augustus only 1 day before the commit you mention.

**Comment by crudelios** [comment]
My commit is 3.0.1.266 so it should be fixed by 282. If it isn't I'll need to check.
